pkg/controller/endpointslice/utils.go

```diff
@@ -133,20 +120,14 @@ func getEndpointAddresses(podStatus corev1.PodStatus, service *corev1.Service) [
 
 	for _, podIP := range podStatus.PodIPs {
 		isIPv6PodIP := utilnet.IsIPv6String(podIP.IP)
-		if isIPv6PodIP == isIPv6Service(service) {
+		if isIPv6PodIP == endpointutil.IsIPv6Service(service) {
```

Comment:
nit, who should have the helper functions, the endpoint or the endpointslice, the former will be deprecated by the later, right?

Reply:
`endpointutil` is shared by both